config/Dockerfiles/ostree-boot-image/default.xml

```diff
@@ -1,5 +1,6 @@
 <network>
   <name>default</name>
+  <uuid>03f5e3d1-164e-4059-9df8-8c1494b81025</uuid>
```

Comment:
do these uuids ever have to change?

Reply:
no - and they could probably even be the same between containers.